3-enrich/stream-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/kinesis/KinesisEnrichApp.scala

```diff
@@ -378,4 +378,12 @@ class KinesisEnrichConfig(config: Config) {
   val maxBackoff = backoffPolicy.getLong("maxBackoff")
 
   val useIpAddressAsPartitionKey = outStreams.hasPath("useIpAddressAsPartitionKey") && outStreams.getBoolean("useIpAddressAsPartitionKey")
+  val partitionKey:Option[String] = if (outStreams.hasPath("partitionKey") && outStreams.getString("partitionKey").isEmpty) {
+    Some(null)
```

Comment:
I don't think this has any benefit over a random UUID since specifying no partition will result in a round robin and hashing UUIDs will result in hopefully the same thing.